app/plugins/shards/shards.html

```diff
@@ -2,22 +2,22 @@
   <div class="page-container">
       <h1 class="cr-page-header">Shards</h1>  
 
-      <div class="cr-panel-block" ng-if="empty">
+      <div class="cr-panel-block" ng-show="empty">
```

Comment:
should this be the exact opposite of the other block?
`ng-show="empty || !shards"`

I know this is kind of unrelated. maybe in a separate commit?

Reply:
i'll make a separate commit for it !